### DISCUSSION TITLE
Creating a new branch 'error: cannot stat'

### DISCUSSION THREAD

**Discussion by theresaperry** [community/community#39187] [New to GitHub]
I keep getting this error whenever I try to create a new branch in GitHub desktop. I also cannot switch between branches. Can anyone assist?

![Screenshot 2022-11-15 160740](https://user-images.githubusercontent.com/111302618/202024204-bf2072a0-12e3-4a49-8388-81c89c869b18.jpg)

**Comment by david-engelmann** [comment]
[You can enable long file names, check out this guide](https://katalonsupport.force.com/katalonhelpcenter/s/article/How-to-fix-Git-Clone-Filename-too-long-Error-in-Windows)!

Most relevant section:
```
To enable long paths in Git Bash, use "git config core.longpaths true"
```
